packages/twenty-front/src/modules/object-record/record-field/hooks/useInitDraftValueV2.ts

```diff
@@ -37,7 +38,10 @@ export const useInitDraftValueV2 = <FieldValue>() => {
           )
           .getValue();
 
-        if (isUndefined(value)) {
+        if (
```

Comment:
why do we have this logic twice? :)

Reply:
right - I think we only want to use useInitDraftValueV2, so I deprecated the usage of initDraftValue from the other hook !

Reply:
(cf https://github.com/twentyhq/twenty/pull/5023)